plugins/inputs/openweathermap/openweathermap.go

```diff
@@ -91,26 +102,16 @@ func (n *OpenWeatherMap) Gather(acc telegraf.Accumulator) error {
 		n.client = client
 	}
 
-	units := n.Units
 	switch n.Units {
-	case "imperial", "standard":
-		break
+	case "imperial", "standard", "metric":
 	default:
-		units = defaultUnits
```

Comment:
Remove the definition for `defaultUnits`

Reply:
ok